source/api.py

```diff
@@ -500,6 +500,14 @@ def isNVDAObject(obj: Any) -> bool:
 	return isinstance(obj, NVDAObjects.NVDAObject)
 
 
+fakeNVDAObject: set[type[Any]] = set()
```

Comment:
Can this get a more descriptive name, and perhaps a docstring?
```suggestion
fakeNVDAObjectClasses: set[type[Any]] = set()
```